docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -74,19 +74,60 @@ Below options are used to configure Azure Event Hubs SDK Client.
+|*spring.cloud.azure.eventhubs*.producer.* | NA | Producer
+configuration options.
+|*spring.cloud.azure.eventhubs*.consumer.* | NA | Consumer
+configuration options.
+|*spring.cloud.azure.eventhubs*.consumer.consumerGroup | NA | Name of the consumer group this consumer is associated with.
+|*spring.cloud.azure.eventhubs*.consumer.prefetchCount | NA | The number of events the Event Hub
+consumer will actively receive and queue locally without regard to whether a receiving operation is currently active.
+|*spring.cloud.azure.eventhubs*.processor.trackLastEnqueuedEventProperties | Boolean |Whether the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
+|*spring.cloud.azure.eventhubs*.processor.initialPartitionEventPosition | Map |The map
+containing the event position to use for each partition if a checkpoint for the partition does not exist in CheckpointStore.
+not the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
+|*spring.cloud.azure.eventhubs*.processor.partitionOwnershipExpirationInterval | Duration |The time duration after which the ownership of partition expires if it's not renewed by the owning processor instance.
+|*spring.cloud.azure.eventhubs*.processor.batch. maxSize| Integer | Max size of message in one
+batch.
+|*spring.cloud.azure.eventhubs*.processor.batch. max-wait-time| Duration | Max time duration for
+one batch of the max size of message.
+|*spring.cloud.azure.eventhubs*.processor.loadBalancing.updateInterval| Duration | The
+interval of load balancing strategy update.
+|*spring.cloud.azure.eventhubs*.processor.loadBalancing.strategy| String | The strategy used by
+event processor for load balancing the partition ownership to distribute the event processing work with other processor instances.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.create-container-if-not-exists |
+Boolean |
+If allowed creating containers if not exists.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.customer-provided-key | String | Base64
+encoded string of the encryption key.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.encryption-scope | String | Encryption
+scope to encrypt blob contents on the server.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.service-version | BlobServiceVersion |
+The versions of Azure Storage Blob supported by this client library.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.blob-name | String | Storage blob name.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.container-name | String | Storage
+container name.
 |===
 
+Notes:
+
+Follow the prefix of `*spring.cloud.azure.eventhubs*.producer.`, all the common properties in the
+parent level
+(connection-string, namespace, domain-name, event-hub-name, custom-endpoint-address) can be
```

Comment:
The common properties include not only the five ones you list, but all entries in the above charts with the prefix begining with "spring.cloud.azure.eventhubs", please modify the notes here as how we discussed. Also this should involve consumer and processor.

Reply:
updated